### ISSUE TITLE
Add DxfDocument.LoadAsync

### ISSUE THREAD

**Issue by bvirkler** [haplokuon/netDxf#159]
Is there a reason there isn't an async version of the `Load` method?  If not, would you be open to a PR adding this?

**Comment by haplokuon** [comment]
No, no special reason, I haven't even thought about it, I just know the basics of parallel programming. In this kind of cases, where a long process is executed, I just create a new thread to avoid locking up the UI. You can try if you wish, while it does not modify too much the loading and writing process I am fine with it, I will still need to look at the changes, thought.
